src/pint/models/binary_ddk.py

```diff
@@ -102,3 +102,11 @@ def validate(self):
                 "DDK model does not accept `SINI` as input. Please"
                 " use `KIN` instead."
             )
+
+        if hasattr(self._parent, "PX"):
```

Comment:
I can't see where `PX` is actually used? Do you have a test that verifies whether it is - for example changing PX and confirming that the residuals from barycentered TOAs change?

Reply:
Here is the usage of DDK

https://github.com/nanograv/PINT/blob/85febde32ded496950797e04ffcfdc81c1923390/src/pint/models/stand_alone_psr_binaries/DDK_model.py#L325-L343

Reply:
Do I understand correctly that this change is supposed to pull the PX value from the parent of the PINT-facing object to the internal standalone binary model? Does that actually happen? Do you test that anywhere?

Reply:
Yes. That is the "test_stand_alone_model_params" is doing